schema-shared.graphql

```diff
@@ -75,6 +75,14 @@ type CollectionPartnership {
   blurb: Markdown!
 }
 
+"""
+Label Type: this is used to specify certain labels for an entity (i.e. collection)
```

Comment:
Please remove `Label Type:` prefix so that it appears more like the rest of our [docstrings](https://studio.apollographql.com/graph/pocket-client-api/schema/reference/objects?variant=current). 

This one is a tough description :joy: Here's an example trying to use another word to describe label. 

```suggestion
A label used to categorize an Entity (e.g. Collection)
```